README.rst

```diff
@@ -101,6 +101,47 @@ asyncio. It requires:
 * PyMongo_ >=3.12,<4
 * Python 3.5+
 
+Optional dependencies:
+
+Motor supports same optional dependencies as PyMongo. Required dependencies can be installed 
+along with Motor.
+
+GSSAPI authentication requires pymongo[gssapi]. The correct
+dependency can be installed automatically along with Motor::
+
+  $ pip install "motor[gssapi]"
+
+similarly,
+
+MONGODB-AWS authentication requires pymongo[aws]::
```

Comment:
The `pymongo[<foo>]` comments are a bit confusing, maybe say something like: "requires the 'aws' extra dependency::"

Reply:
removed PyMongo's reference and instead added inline highlighting to dependency names.